### DISCUSSION TITLE
Volumes created with Block frontend fail to attach

### DISCUSSION THREAD

**Discussion by jej2003** [longhorn/longhorn#3780] [General]
I'll start by saying I'm new to K8s and longhorn as well.  I have setup a simple rke2 single node setup, installed longhorn following the documentation and everything appears to be ok.  When creating a volume through the longhorn UI if I select Block frontend and try to attach it the volume state bounces from "Attaching" to "Detaching" and eventually fails.  If I instead specify the iSCSI frontend it can be attached as expected.  If I select maintenance when attaching it, it also attaches fine.  I am struggling to figure out where to start debugging the issue, can anyone point me in the right direction?

**Comment by jej2003** [comment]
looks like the issue was related to environmental issues.  After installing the right version of rke2 everything seems good

**Comment by PhanLe1010** [comment]
Discussed in the ticket https://github.com/longhorn/longhorn/issues/3781
